js/constants/apiUrls.js

```diff
@@ -0,0 +1,15 @@
+const mainUrl = 'https://bootcamp-api.codeit.kr/api/';
```

Comment:
변수명은 대문자로 작성되는게 좋을 것 같아요!
```suggestion
const MAIN_URL = 'https://bootcamp-api.codeit.kr/api/';
```

Reply:
그리고 mainUrl보다는 baseUrl 혹은 apiBaseUrl과 같은 표현을 일반적으로 쓰는 것 같아요!
```suggestion
const BASE_URL = 'https://bootcamp-api.codeit.kr/api/';
```

Reply:
그리고 보통 회사에서 개발을 할 때는 배포 환경을 2가지로 가져가는데요!
staging은 회사 내부에서 테스트 하는 환경,
production은 실제로 사용자들이 서비스를 이용하는 환경을 의미해요.
- staging
- production

프로젝트 코드를 어떤 환경에 배포하냐에 따라서 그 환경에 맞는 서버의 주소(API_BASE_URL)를 적용해줘야 하는데요! 이것은 나중에 `환경변수`를 이용해서 처리해주게 되어요! 
나중에 이 부분도 학습해주시면 좋을 것 같아요!